keylime.conf

```diff
@@ -34,12 +34,12 @@ rsa_keyname = tci_rsa_key
 # Enable mTLS communication between agent, verifier and tenant.
 # Details on why setting it to "False" is generally considered insecure can be found
 # on https://github.com/keylime/keylime/security/advisories/GHSA-2m39-75g9-ff5r
-mtls_cert_enabled = True
```

Comment:
I think the changes to the configuration file other than the URL for the documentation was not intentional, right?

Reply:
@mpeters I have the same question. Shouldn't it be on a different PR?

Reply:
yes, sorry accidentally included my local dev changes without looking